lib/concourse-binary-builder.rb

```diff
@@ -232,8 +232,8 @@ def commit_and_rsync(in_dir, out_dir, file)
 
     Dir.chdir(in_dir) do
 
-      GitClient.set_global_config('user.email', 'cf-buildpacks-eng@pivotal.io')
-      GitClient.set_global_config('user.name', 'CF Buildpacks Team CI Server')
+      GitClient.set_global_config('user.email', 'cfcibot@suse.de')
```

Comment:
The email address is `cf-ci-bot@suse.de`. `cfcibot` is the user name on DockerHub.